libs/sof/lk_tracker.cpp

```
@@ -160,6 +157,13 @@ bool LKFeatureTracker::trackPoint(const GradientPyramidT& prevFrameGradPyramid,
           xy2 += v;
         }
 
+        if (isConverged) {
```

Clang (14.0.6)

[note] +4, including nesting penalty of 3, nesting level increased to 4

(clang)